components/examples/window/src/main.ts

```diff
@@ -0,0 +1,178 @@
+import van from "vanjs-core"
+import { FloatingWindow, Tabs } from "vanjs-ui"
+
+export type CSSPropertyBag = Record<string, string | number>
+
+const toStyleStr = (style: CSSPropertyBag) =>
+  Object.entries(style).map(([k, v]) => `${k}: ${v};`).join("")
+
+const { button, div, h1, p, a, b, code, pre, span, style } = van.tags
+const example1 = () => {
+  const closed = van.state(false)
+  const x = van.state(100)
+  const y = van.state(100)
+  const width = van.state(300)
+  const height = van.state(200)
+
+  van.add(document.body, FloatingWindow({
+    title: "Example Window 1",
+    closed,
+    x,
+    y,
+    width,
+    height
+  },
+    div({ style: "padding: 1rem;" },
+      p("Hello, World!"),
+      div({ style: "display: flex; justify-content: center;" },
+        button({ onclick: () => closed.val = true }, "Close Window"),
+      ),
+    )
+  ))
+}
+
+const example2 = () => {
+  const closed = van.state(false)
+  const x = van.state(150)
+  const y = van.state(150)
+  const width = van.state(300)
+  const height = van.state(200)
+
+  van.add(document.body, FloatingWindow({
+    title: "Example Window 2",
+    closed,
+    x,
+    y,
+    width,
+    height,
+    closeCross: true,
+    headerStyleOverrides: {
+      "background-color": "lightblue",
+    }
+  },
+    div({ style: "display: flex; justify-content: center;" },
+      p("This is another floating window!"),
+    )
+  ))
+}
+
+const example3 = () => {
+  const closed = van.state(false)
+  const x = van.state(200)
+  const y = van.state(200)
+  const width = van.state(500)
+  const height = van.state(300)
+
+  van.add(document.body, FloatingWindow({
+    title: "Example Window 3",
+    closed,
+    x,
+    y,
+    width,
+    height,
+    closeCross: true,
+    windowStyleOverrides: {
+      "background-color": "lightgray",
+      padding: "0px",
+    },
+    headerStyleOverrides: {
+      display: "none",
+    },
+
+  },
+    div(
+      span({
+        style: toStyleStr({
+          position: "absolute",
+          top: "0px",
+          right: "0px",
+          padding: "0.5rem",
+          cursor: "pointer",
+          height: "1.2rem",
+        }),
+        class: "custom-cross",
+        onclick: () => closed.val = true,
+      }, "×"),
```

Comment:
Is this element duplicate with `closeCross: true` above?

Reply:
Let remove the `closeCross: true` line since it doesn't take effect. Otherwise it might confuse the reader (they might think it's `closeCross: true` control whether the cross button to show or not).

Reply:
Oh, my bad, it was a remain of a paste. Yes it's useless